Automattic/pocket-casts-ios
# Fix up next navigation items

Fixes #

Please include a summary of the changes and the related issue.

## To test

- Start the app
- Clean UpNext items ( long press on the mini-player image and choose Close And Clear Up Next)
- Check that the navigation bar in clean of items ( Select )
- Go to the podcasts tab
- Open a podcast and add some items to the UpNext queue
- Switch to the UpNext tab
- Check that the navigation bar has the Select option
- Tap Select
- Choose some items and while they are selected
- Clean UpNext items ( long press on the mini-player image and choose Close And Clear Up Next)
- Check that the navigation bar updates correctly, it should show any actions

## Checklist
